openpype/hosts/resolve/api/lib.py

```diff
@@ -243,10 +246,13 @@ def get_media_pool_item(fpath, root: object = None) -> object:
     return None
 
 
-def create_timeline_item(media_pool_item: object,
-                         timeline: object = None,
-                         source_start: int = None,
-                         source_end: int = None) -> object:
+def create_timeline_item(
+        media_pool_item: object,
+        source_start: int,
+        source_end: int,
+        timeline_in: int,
+        timeline: object = None
+) -> object:
```

Comment:
Why do we now require these arguments? I'd expect the lib function to still just allow to the relevant entry if no source start is specificied, etc.

I'd also expect the arguments ordered:
```suggestion
def create_timeline_item(
        media_pool_item: object,
        timeline: object = None,
        timeline_in: int = None,
        source_start: int = None,
        source_end: int = None,
) -> object:
```

So that to just add a clip to a timeline would be: `create_timeline_item(item, timeline)`

Reply:
Yeah, you are right, this lost the backward compatibility..

Reply:
and other related changes here 435408dc142cb934424333257f8092db37974b01